{"name": "Параметризованная проверка невозможности авторизации при передаче неверного логина или пароля", "status": "passed", "description": "Создаем курьера, пытаемся авторизоваться передавая неверный логин или пароль,                         проверяем что код ответа: 404, текст ответа: \"Учетная запись не найдена\"", "parameters": [{"name": "incorrect_argument", "value": "'password'"}], "start": 1710001680274, "stop": 1710001680907, "uuid": "41c371ba-51f3-4c13-bf54-a3681f70b88a", "historyId": "8eeac7e6dbc793f67f7ca3bbffd8b667", "testCaseId": "494d196ed4678e26fab0bc9442c7c320", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_courier_with_incorrect_login_or_password_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-1VNC7DB"}, {"name": "thread", "value": "34604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}